tests/Analytics/AnalyticsTest.php

```diff
@@ -15,15 +15,15 @@
 namespace Utopia\Tests;
 
 use PHPUnit\Framework\TestCase;
-use Utopia\Analytics\GoogleAnalytics;
-
+use Utopia\Analytics\Adapter\GoogleAnalytics;
 class AnalyticsTest extends TestCase
 {
     public $ga;
+    public $plausible;
 
     public function setUp(): void
     {
-        $this->ga = new GoogleAnalytics("UA-XXXXXXXXX-X", "test");
+        $this->ga = new GoogleAnalytics("UA-132779463-1", "test");
```

Comment:
Are we sure about this change? Aren't we exposing some internal IDs?

Reply:
I've rolled back this change, it was just for testing